app/assets/stylesheets/partials/_pagination.scss

```diff
@@ -0,0 +1,47 @@
+
+.ceds-pagination {}
+
+.ceds-pagination__summary {
+  @extend .govuk-grid-column-one-half;
+}
+
+.ceds-pagination__controls {
+  @extend .govuk-grid-column-one-half;
+  @extend .align-right;
+}
+
+.ceds-pagination__items {
+  display: inline;
+}
+
+// Can it be changed to class?
```

Comment:
It still in single block would be enough.
Is just item class as selectro is not enough ?
Maybe would be worth to merge this declaration on next one ?